commentary.md

```diff
@@ -84,15 +84,39 @@ Eventually the `Syntax` gets simplified into a core language which we can evalua
 
 `Core` expressions consist of function applications, pattern-matching, and constructors for the language's primitive types. Those primitive types include functions, booleans, signals, reified `Syntax` objects, and "macro actions". A macro action is a monadic block performing zero or more side-effects. These side-effects do not occur during evaluation, but rather during expansion, so we'll cover macro actions in more details in the `Expanding` section.
 
-TODO: what happened to strings? `Syntax` supports booleans, strings, and signals, but of those, `Core` only supports booleans and signals. Maybe we're instead supposed to manipulate a reified `Syntax` object holding a string?
```

Comment:
This TODO about strings has nothing to do with the rest of the changes, it's just an outdated TODO, as Core _does_ have a representation for strings now. There are also a bunch of places in the commentary which still refer to signals, but I haven't fixed them yet.